atom/browser/api/atom_api_cookies.h

```diff
@@ -53,6 +53,7 @@ class Cookies : public mate::TrackableObject<Cookies>,
   void Remove(const GURL& url, const std::string& name,
               const base::Closure& callback);
   void Set(const base::DictionaryValue& details, const SetCallback& callback);
+  void FlushStore(const base::Closure& callback);
```

Comment:
Could we add a test for this to `api-session-spec.js` that just verifies the callback is invoked, you don't need to assert more than that, but just to have basic coverage of this API.